ReleaseNotes.md

```diff
@@ -1,3 +1,12 @@
+# Release_2020.09.1
+(Changes relative to Release_2020.03.1)
+
+## Deprecations
```

Comment:
I think it makes sense to have this in the `2020.03.1` release notes and then remove it for `2020.09.1`

Reply:
Whoops, sorry, I don't even now which month I'm living in anymore...

Reply:
yeah, I undestand how that happens. :-)